hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/RolloutGroupManagement.java

```diff
@@ -150,4 +150,14 @@
      */
     @PreAuthorize(SpringEvalExpressions.HAS_AUTH_ROLLOUT_MANAGEMENT_READ)
     RolloutGroup findRolloutGroupWithDetailedStatus(@NotNull Long rolloutGroupId);
+
+    /**
+     * Count targets of rollout group.
+     * 
+     * @param rolloutGroup
+     *            the rollout group for the count
+     * @return the target rollout group count
+     */
+    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_ROLLOUT_MANAGEMENT_READ)
+    Long countTargetsOfRolloutsGroup(@NotNull RolloutGroup rolloutGroup);
```

Comment:
I'd use the `id` of the `RolloutGroup` instead of the `RolloutGroup` object like `Long countTargetsOfRolloutsGroup(@NotNull Long rolloutGroupId);`